tiledb/api/c_api/dimension/dimension_api.cc

```diff
@@ -32,8 +32,10 @@
 
 #include "../../c_api_support/c_api_support.h"
 #include "../filter_list/filter_list_api_internal.h"
+#include "../string/string_api_external.h"
```

Comment:
Is this header needed? In almost all cases it's not the right thing to use one C API function to implement another one.